src/main/java/org/c4sg/controller/SkillController.java

```diff
@@ -39,15 +42,47 @@
     @RequestMapping(value="/user",produces = {"application/json"}, method = RequestMethod.GET)
     @ApiOperation(value = "Get skills for an user by id", notes = "Returns a collection of skills for an user by id")
     public List<String> getSkillsForUser(@ApiParam(value = "ID of user to return", required = true)
-    									 @RequestParam Integer id) {
+    					 @RequestParam Integer id) {
         return skillService.findSkillsForUser(id);
     }
     
     @CrossOrigin
     @RequestMapping(value="/project",produces = {"application/json"}, method = RequestMethod.GET)
     @ApiOperation(value = "Get skills for a project by id", notes = "Returns a collection of skills for a project")
     public List<String> getSkillsForProject(@ApiParam(value = "ID of project to return", required = true)
-    									 @RequestParam Integer id) {
+    					    @RequestParam Integer id) {
         return skillService.findSkillsForProject(id);
     }
-}
\ No newline at end of file
+    
+    @CrossOrigin
+    @RequestMapping(value="/user/addSkills", method = RequestMethod.POST)
```

Comment:
according to the REST pattern you should replace the word `addSkills` by `skills`